queries.sql

```diff
@@ -82,3 +82,178 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
   4 | Devimon | 2017-05-12    |               5 | t       |        11
 (2 rows)
 */
+
+
+/* /////////////////////////////////////////////////////////////////// */
+
+START TRANSACTION;
+
+UPDATE animals SET species = 'unspecified';
```

Comment:
As per the project requirements, please add a statement right after updating the animals table to verify that a change was made🙏

Reply:
- [ ] Kindly address this part of the comment made by the previous code reviewer. Kindly check the highlighted screenshot to be able to understand the comments fully
![Screenshot from 2023-07-12 09-42-09](https://github.com/rachaouldbabaali/vet-clinic-db/assets/46388113/52881bd4-f963-4fe4-91b5-7839e0824c38)